docs/installation.md

```diff
@@ -89,36 +103,44 @@ OS Specific steps are listed first, the [Common](#common) section below is neces
 !!! Note
     Python3.6 or higher and the corresponding pip are assumed to be available.
 
-### Linux - Ubuntu 16.04
+=== "Ubuntu 16.04"
+    #### Install necessary dependencies
 
-#### Install necessary dependencies
+    ```bash
+    sudo apt-get update
+    sudo apt-get install build-essential git
+    ```
 
-```bash
-sudo apt-get update
-sudo apt-get install build-essential git
-```
+=== "RaspberryPi/Raspbian"
+    The following assumes the latest [Raspbian Buster lite image](https://www.raspberrypi.org/downloads/raspbian/) from at least September 2019.
+    This image comes with python3.7 preinstalled, making it easy to get freqtrade up and running.
 
-### Raspberry Pi / Raspbian
+    Tested using a Raspberry Pi 3 with the Raspbian Buster lite image, all updates applied.
 
-The following assumes the latest [Raspbian Buster lite image](https://www.raspberrypi.org/downloads/raspbian/) from at least September 2019.
-This image comes with python3.7 preinstalled, making it easy to get freqtrade up and running.
+    ``` bash
+    sudo apt-get install python3-venv libatlas-base-dev
+    git clone https://github.com/freqtrade/freqtrade.git
+    cd freqtrade
 
-Tested using a Raspberry Pi 3 with the Raspbian Buster lite image, all updates applied.
+    bash setup.sh -i
+    ```
 
-``` bash
-sudo apt-get install python3-venv libatlas-base-dev
-git clone https://github.com/freqtrade/freqtrade.git
-cd freqtrade
+    !!! Note "Installation duration"
+        Depending on your internet speed and the Raspberry Pi version, installation can take multiple hours to complete.
 
-bash setup.sh -i
-```
+    !!! Note
+        The above does not install hyperopt dependencies. To install these, please use `python3 -m pip install -e .[hyperopt]`.
+        We do not advise to run hyperopt on a Raspberry Pi, since this is a very resource-heavy operation, which should be done on powerful machine.
 
-!!! Note "Installation duration"
-    Depending on your internet speed and the Raspberry Pi version, installation can take multiple hours to complete.
+=== "Anaconda"
+    Freqtrade can also be installed using Anaconda (or Miniconda).
 
-!!! Note
-    The above does not install hyperopt dependencies. To install these, please use `python3 -m pip install -e .[hyperopt]`.
-    We do not advise to run hyperopt on a Raspberry Pi, since this is a very resource-heavy operation, which should be done on powerful machine.
+    !!! Note
+        This requires the [ta-lib](#1-install-ta-lib) C-library to be installed first. See below.
+    
+    ``` bash
+    conda env create -f environment.yml
+    ```
```

Comment:
This section is kindof wrong here (the header is "necessary dependencies").

Instead, the conda approach installs "everything" - including freqtrade, so all above "common" steps are not necessary when taking this approach (should not be necessary - been a while since conda installation has been tested i guess).

Reply:
Moved Anaconda to the original spot.